Create note for installing Software on GCP (#1879)
astronomer/docs: software/install-gcp-standard.md

* Permission to create and modify resources on Google Cloud Platform
* Permission to generate a certificate (not self-signed) that covers a defined set of subdomains

> :::intro

Reviewer: ```suggestion
:::info
```

> When Astronomer Software is installed on a GCP GKE Dataplane V2 cluster, the interaction between the Astronomer Nginx ingress controller and Cilium can cause dropped connections, dropped packets, and intermittent 504 timeout errors when accessing the Astronomer UI or Houston API.

Reviewer 2: I think we should qualify this as being a bug in something other than our software. As it is currently written it seems as though only Astronomer "Software" would have this issue, but it's not our bug.

> When Astronomer Software is installed on a GCP GKE Dataplane V2 cluster, the interaction between the Astronomer Nginx ingress controller and Cilium can cause dropped connections, dropped packets, and intermittent 504 timeout errors when accessing the Astronomer UI or Houston API.

Author: ```suggestion
There is a known bug on GCP GKE Dataplane V2 clusters that affects Astronomer Software installations. When Astronomer Software is installed on a GCP GKE Dataplane V2 cluster, the interaction between the Astronomer Nginx ingress controller and Cilium can cause dropped connections, dropped packets, and intermittent 504 timeout errors when accessing the Astronomer UI or Houston API.
```

> There is a known bug with installing Astronomer Software on a GCP GKE Dataplane V2 cluster which is related to the Astronomer's Nginx ingress controller and how it interacts with Cilium. This issue results in dropped connections, dropped packets, and intermittent 504 timeout errors when accessing the Astronomer UI or Houston API.

Reviewer: ```suggestion
When Astronomer Software is installed on a GCP Google Kubernetes Engine (GKE) Dataplane V2 cluster, an interaction between the Astronomer Nginx ingress controller and Cilium can cause dropped connections, dropped packets, and intermittent 504 timeout errors when accessing the Astronomer UI or Houston API.
```

> There is a known bug with installing Astronomer Software on a GCP GKE Dataplane V2 cluster which is related to the Astronomer's Nginx ingress controller and how it interacts with Cilium. This issue results in dropped connections, dropped packets, and intermittent 504 timeout errors when accessing the Astronomer UI or Houston API.

Reviewer 3: ```suggestion
There is a known bug with installing Astronomer Software on a GCP GKE Dataplane V2 cluster which is related to the Astronomer's Nginx ingress controller and how it interacts with Cilium. This issue presents symptoms such as dropped connections, dropped packets, and intermittent 504 timeout errors when accessing the Astronomer UI or Houston API.
```

> Install Astronomer Software to a GKE Dataplane V1 cluster to avoid these issues. 

Reviewer: ```suggestion
To prevent these issues, Astronomer recommends installing Astronomer Software on a GKE Dataplane V1 cluster. 
```

Reviewer 3: I think 'avoid' is more accurate than 'prevent'
